docs/hacking.md

```diff
@@ -22,6 +25,26 @@ This will also run code coverage and send the report to [Coveralls](http://cover
 
 Note that code coverage reports are also [browseable on Coveralls](https://coveralls.io/r/mozilla-services/kinto.js).
 
+### Functional tests
+
+In order to run the functional tests (which are part of the test-suite), you
+will need to install Kinto locally and export the location of `pserve`
+accordingly.
+
+The best way to install kinto is in a virtual environment and activate it.
+
+    $ virtualenv venv
+    $ ./venv/bin/pip install kinto
+    $ source ./venv/bin/activate
+
+In case you don't want to activate your venv each time, you can also export the
+location of the `pserve` executable in the `KINTO_PSERVE_EXECUTABLE`
+environment variable.
```

Comment:
Actually it may be simpler to run the test command prefixed with the env var declaration:

```
$ KINTO_PSERVE_EXECUTABLE=/Users/niko/Sites/kinto/.venv/bin/pserve npm test
```